src/ui/Pager/Pager.ts

```diff
@@ -146,6 +144,26 @@ export class Pager extends Component {
     element.appendChild(this.list);
   }
 
+  /**
+   * The current page (1-based index).
+   */
+  public get currentPage(): number {
+    return this._currentPage;
+  }
+
+  public set currentPage(value: number) {
+    let sanitizedValue = value;
+
+    if (isNaN(value)) {
+      this.logger.warn(`Unable to set pager current page to an invalid value: ${value}. Resetting to 1.`);
+      sanitizedValue = 1;
+    }
+
+    sanitizedValue = Math.round(sanitizedValue);
```

Comment:
Out of curiosity, why `Math.round` instead of floor ? :D

I would think "51 first results out of 100" means page 1,

Reply:
My intuition would be to use a `Math.floor` or `parseInt` as well.
e.g
resultNumber = 6
resultsPerPage = 10
pageNumber = 6/10 = 0.6
roundedPageNumber = 1
flooredPageNamber = 0

Since the pageNumber is a one-based index, we would need to add 1:
roundedPageNumber = 2
flooredPageNumber = 1

I think result number 6 should be on page 1? So the floor would give the correct value?